roles/edpm_libvirt/templates/qemu.conf.j2

```diff
@@ -2,7 +2,6 @@ max_files = 32768
 max_processes = 131072
 vnc_tls = 0
 vnc_tls_x509_verify = 0
-default_tls_x509_verify = 1
 nbd_tls = 0
```

Comment:
is this needed for adoption from computes with NBD TLS enabled?

Reply:
Based on https://bugzilla.redhat.com/show_bug.cgi?id=1879477 it is enabled by default since libvirt 6.10 so I think we can remove it from here. I don't think it effects adoption as I expect that this is enabled there as well.

Reply:
It is also added conditionally below. I think it is OK to remove it from both places as the default is the same.

Reply:
was referring to `nbd_tls = 0`

Reply:
sorry my bad. Honestly I'm out of my depth here. I assume if we keep this here then we need to add nbd_tls = 1 below in the conditional block to ensure that nbd uses tls when tls is enabled.